edb/server/pgcon/pgcon.pyx

```diff
@@ -1635,6 +1639,14 @@ cdef class PGConnection:
         while self.buffer.take_message():
             if not self.parse_notification():
                 mtype = self.buffer.get_message_type()
+                if mtype == b'E':  # ErrorResponse
+                    er_cls, fields = self.parse_error_message()
+                    msg = POSTGRES_SHUTDOWN_ERR_CODES.get(fields['C'])
+                    if msg:
+                        msg = fields.get('M', msg)
```

Comment:
hm, this variable is unused, no? Also, what's the plan on handling the shutdown?

Reply:
Oh sorry, this hunk was moved to #2574 - I'll update this PR and remove this change.